packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv/ai/__init__.py

```diff
@@ -36,6 +36,14 @@ class SpanAttributes:
     # Deprecated
     TRACELOOP_CORRELATION_ID = "traceloop.correlation.id"
 
+    # Watson/genai LLM
+    LLM_DECODING_METHOD = "decoding_method"
```

Comment:
I'd use `llm.watson.decoding_method` in line with what we're doing in https://github.com/open-telemetry/semantic-conventions/pull/639/files#diff-e352d0862431e100006cb429634845d199d25ad43c6d09c14e08267c7b73f5b4
(Same for others)

Reply:
Done